### PR TITLE
fix: docusaurus config

### PR DESCRIPTION
**Pull request by vojtechsimetka**
Fixes some local warning, adds correct repo URLs and fixes typerror in lunar search when run locally (see https://github.com/lelouch77/docusaurus-lunr-search/issues/33 )

### CODE REVIEW

**Comment by significance** [comment]
@vojtechsimetka looks good as far as i can see but not building - happy to approve or let me know how best to check otherwise

**Comment by vojtechsimetka** [comment]
@significance that is because the fix for that was in a different branch. Now I've merged it and it works as you can see here https://github.com/ethersphere/bee-js.github.io/pull/1/checks?check_run_id=1821256237

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
   onBrokenLinks: 'warn',
```
Comment (significance) [inline docusaurus.config.js:8]: above, url should be `bee-js.ethswarm.org`, note that we need to amend this (and base_url) automatically when we build for Bee in github actions

**Review by significance — CHANGES_REQUESTED** [review]
just small change as per comment as far as i can see, otherwise looks good!

**Review by significance — APPROVED** [review]
lgtm! 😀

**Pull request merged**